pkg/windows/windows.go

```diff
@@ -139,6 +139,15 @@ const (
 	wicdKubeconfigPath = K8sDir + "\\wicd-kubeconfig"
 	// TrustedCABundlePath is the location of the trusted CA bundle file
 	TrustedCABundlePath = remoteDir + "\\ca-bundle.crt"
+	// GetHostnameFQDNCommand is the PowerShell command to get the FQDN hostname of the Windows instance
+	GetHostnameFQDNCommand = "$output = Invoke-Expression 'ipconfig /all'; " +
+		"$hostNameLine = ($output -split '`n') | Where-Object { $_ -match 'Host Name' }; " +
+		"$dnsSuffixLine = ($output -split '`n') | Where-Object { $_ -match 'Primary Dns Suffix' }; " +
+		"$hostName = ($hostNameLine -split ':')[1].Trim(); " +
+		"$dnsSuffix = ($dnsSuffixLine -split ':')[1].Trim(); " +
+		"if (-not $dnsSuffix) { return $hostName }; " +
```

Comment:
same question, if there is no dnsSuffix then will the output of this be equivalent to `vm.Run("hostname", true)`

Reply:
correct, and the evidence in all e2e test job that passed, none are using a windows instance that is part of a domain controller, so dnsSuffix is empty and returns just the hostName